### ISSUE TITLE
AK magazine switching deletes two classes of magazine from inventory

### ISSUE THREAD

**Issue by s4098055** [toadie2k/NIArms#76]
G'day all,

It appears that whatever script the 5.45mm AK series is using to switch weapon models is not restocking two classes of magazine when the weapon model changes: 

hlc_30rnd_545x39_t_ak (standard AK tracer mag)
hlc_30rnd_545x39_ep_ak (7N10 advanced 'peneterator', heh)

This means your inventory will be emptied of these two types of magazine if you change the weapon model by loading an RPK74 45-rounder or RPK12 60-rounder, for example.

Reproduction is fairly simple; jump into the arsenal, arm yourself with a 5.45mm-accepting AK, give yourself any of the above two magazine classes and an RPK magazine. Watch as your inventory empties on the change of the weapon model.

Before: https://gyazo.com/5c3b86a964fd563ec2a0f5592dd3d3e5

After: https://gyazo.com/bf329c8ab2add116243932a6002ad0d1

Tested in A3/HLC/CBA (the above pictures were from the initial testing, using a different modset).

...also, is "peneterator" a legitimate spelling error or some sort of inside joke I'm not privy to? 😸

**Comment by toadie2k** [comment]
Sadly, it's a spelling error, and one that noone has caught for probably years

**Comment by toadie2k** [comment]
@Torndeco  - Can confirm whether #72 solved this?

**Comment by Torndeco** [comment]
Pull Request #72 & #73 will solve this issue.
Have been running it on public live servers (custom addon to override the code), for awhile now

**Comment by toadie2k** [comment]
Cool, just needed to double check to make sure this wasn't some new issue, as I couldn't replicate on my current build.
